src/command/backup/file.c

```diff
@@ -227,8 +227,11 @@ backupFile(
                         storageReadIo(read)), cipherBlockNew(cipherModeEncrypt, cipherType, BUFSTR(cipherPass), NULL));
             }
 
-            // Setup the repo file for write
-            StorageWrite *write = storageNewWriteP(storageRepoWrite(), repoPathFile, .compressible = compressible);
+            // Setup the repo file for write. There is no need to write the file atomically (e.g. via a temp file on Posix) because
+            // checksums are tested on resume after a failed backup. The path does not need be synced for each file because all
```

Comment:
`The path does not need be synced` --> add "to": `The path does not need to be synced`

Reply:
Fixed in 5b4e95ed14b72d.